lib/developer_portal/app/controllers/developer_portal/admin/account/passwords_controller.rb

```diff
@@ -12,11 +12,11 @@ class DeveloperPortal::Admin::Account::PasswordsController < ::DeveloperPortal::
   def create
     return redirect_to_request_password('Bot protection failed.') unless bot_check({ flash: false })
 
-    user = @provider.buyer_users.find_by_email(params[:email])
-    return redirect_to_request_password('Email not found.') unless user
+    email = params[:email]
+    user = @provider.buyer_users.find_by(email: email)
+    user&.generate_lost_password_token!
 
-    user.generate_lost_password_token!
-    flash[:notice] = 'A password reset link has been emailed to you.'
+    flash[:notice] = "A password reset link has been emailed to: #{email}."
```

Comment:
```suggestion
    flash[:notice] = "A password reset link will be emailed if a user exists registered with #{email}."
```

Reply:
Sounds reasonable, but I just followed the example we have for the admin portal password reset: https://github.com/3scale/porta/blob/cfc866f9c3890afe59d67b69df66dd0a0bd061d2/config/locales/en.yml#L397-L398

Reply:
Also, in this case this message appears in a flash message, so we need to keep it pretty short. Let me see...

Reply:
I think I prefer the short version

Reply:
Actually, I think the longer version doesn't look too bad:

![Screenshot from 2024-06-27 11-31-33](https://github.com/3scale/porta/assets/1270328/86656f94-2dba-4e6c-be2d-601bd82d25fb)

(I tweaked a bit the suggestion)